08-quiz-time.md

```diff
@@ -191,7 +197,7 @@ Choices:
 
 
 ```
-PUT YOUR ANSWER HERE
+Yes

```

Comment:
Incorrect You need to add a unique 'key' prop if you have multiple nested components

Reply:
Thank for you correction ❤️.